File: test/lua/antipatern.lua
return function (params) {
  local __expr0__ = ""
                             
if params.c then
                               
  
                              
  __expr0__ = "fail"
                           
end


params.d = params.a - params.b

local __expr1__ = ""
                             
if params.d then
                               
  
                              
  __expr1__ = "good"
                           
end

  return ((params.c = params.a + params.b, params.c))..__expr0__..__expr1__
}

File: test/js/antipatern.js
module.exports = function (params) {
  var __expr0__ = "";
  if (params.c) {
    __expr0__ = "fail"
  };
  params.d = params.a - params.b;
  var __expr1__ = "";
  if (params.d) {
    __expr1__ = "good"
  }
  return ((params.c = params.a + params.b, params.c)) + __expr0__ + __expr1__
}